.github/workflows/frontend.yml

```diff
@@ -49,7 +49,7 @@ jobs:
           restore-keys: |
             ${{ runner.os }}-zeppelin-
       - name: Install application
-        run: ./mvnw -B install -DskipTests -DskipRat -pl ${INTERPRETERS} -Phadoop2 -Pscala-2.11
+        run:  ./mvnw clean install -pl zeppelin-web -B -Pintegration -Pspark-scala-2.11 -Pspark-2.4 -Phadoop2 -DskipTests -DskipRat -am
```

Comment:
You should use the same profiles that are used for testing. If the profiles are not sufficient for testing, they must be adjusted so that the profiles are the same during installation and testing.
The order of the Maven parameters should be the same for better overview. The parameter `-B` should be at the end. This simplifies the local testing very much.